sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -366,22 +366,6 @@ def test_sdf(self):
 
```

Comment:
rename test_sdf to test_sdf_with_sdf_initiated_checkpointing

might also want to reorder this test so that it appears after the other one.